doc/man/salt.7

```diff
@@ -7095,7 +7095,7 @@ on a large number of minions.
 .INDENT 0.0
 .INDENT 3.5
 Rather than altering this configuration parameter, it may be advisable to
-use the \fBfileserver.clear_list_cache\fP runner to clear these caches.
+use the \fBfileserver.clear_file_list_cache\fP runner to clear these caches.
```

Comment:
This is fine, but a note for the future that the man pages are autogenerated based on the changes in the docs. Updating the man pages is not necessary.